packages/units/src/range.js

```diff
@@ -13,35 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-/**
- * Linear interpolation function.
- *
- * Takes a normalized progress value and a range in the form of a tuple
- * and returns the interpolated progress within the given range.
- *
- * @param {number} progress - value between 0 and 1
- * @param {[number, number]} range - tuple that dictates a range between 2 numbers
- * @return {number} value within given range
- */
 export const lerp = (progress, range) =>
   (1 - progress) * range[0] + progress * range[1];
 
-/**
- * Takes a value and a range in the form of a tuple and clamps
- * the value to that range.
- *
- * @param {number} v - value to be clamped
- * @param {[number, number]} range - range in the form of a tuple
- * @return {number} - number within range
- */
-export const clamp = (v, range) => {
```

Comment:
Note that this function had a different signature than the new one below

Reply:
There were two versions of `clamp` defined in three different packages, one can accept an array ( `[number, number]` ) and other accepts object (`{MIN: number, MAX: number}`)

https://github.com/GoogleForCreators/web-stories-wp/blob/0f12659b6a277de64da7f65922aa5eb666a6614c/packages/story-editor/src/utils/clamp.js#L17

https://github.com/GoogleForCreators/web-stories-wp/blob/e00e50da7a02d956b993f4038ca1c15d46eda3ef/packages/dashboard/src/utils/clamp.js#L17-L22

https://github.com/GoogleForCreators/web-stories-wp/blob/e00e50da7a02d956b993f4038ca1c15d46eda3ef/packages/animation/src/utils/range.js#L38-L43

So I just combined those two versions in one and moved to `units`. Does it bother you that `range` can accept both object and array?

Reply:
> Does it bother you that `range` can accept both object and array?

A little bit, yes :-) Can we settle on one? Perhaps the object, as it's more explicit

Reply:
https://github.com/GoogleForCreators/web-stories-wp/pull/10717/commits/58bc976f8afd04a765180d0cb592abe278dff745